docs/user-guide/command-line-interface.md

```diff
@@ -39,6 +39,7 @@ Options:
   --no-color                  Disable color in piped output
   -o, --output-file path::String  Specify file to write report to
   --quiet                     Report errors only - default: false
+  --max-warnings Number       Maximum warning count before eslint exits with an error (or -1 for no limit)
```

Comment:
Curious, what would be the use case for running with `--max-warnings` but then turning it back off by setting to `-1`?